### ISSUE TITLE
UI: noVNC encountered an error - Cannot set property 'scaleViewpoer' of undefined

### ISSUE THREAD

**Issue by GabrielBrascher** [apache/cloudstack#5068]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
4.15.0.0
~~~

##### CONFIGURATION

New UI and noVNC enabled.

##### OS / ENVIRONMENT

N/A

##### SUMMARY

This issue is being hard for me to reproduce, but it happened sometimes.

The issue happens when resizing the console (noVNC) window. Error:

```
noVNC encountered an error: 

Uncaught TypeError: Cannot set property 'scaleViewpoer' of undefined <https URL>
```

I could not find anything concrete about what is triggering this issue. This exception points to line [systemvm/agent/noVNC/app/ui.js#L1216:30](https://github.com/apache/cloudstack/blob/280c13a4bb103dd748ec304bfe0714a148c24602/systemvm/agent/noVNC/app/ui.js#L1216). However, it happens of noVNC console URL work many times in a row (same network, browser, etc) and suddenly there is an issue.

It is important to note that the URL for the console, when inspecting the page, always works.

In summary:
1. URL works always (sometimes failed when resizing, but opening URL on another window brings the console)
2. DNS is working
3. Console System VM is working fine and presents properly the VM's console

I am not sure if this is a problem on the UI, or at another point, but the exception is raised at [UI.rfb.scaleViewport = this.fullScreen](https://github.com/apache/cloudstack/blob/280c13a4bb103dd748ec304bfe0714a148c24602/systemvm/agent/noVNC/app/ui.js#L1216):
```
    toggleFullscreen() {
        this.fullScreen = !this.fullScreen
        UI.rfb.scaleViewport = this.fullScreen
        UI.updateFullscreenButton(this.fullScreen);
        UI.focusOnConsole();
    },
```

**Comment by weizhouapache** [comment]
@GabrielBrascher this should have been fixed by #4833

**Comment by GabrielBrascher** [comment]
Ah might be the case. Thanks for the quick response @weizhouapache.

I will test it with the changes from #4833. If it persists I will re-open this issue.

**Comment by rhtyd** [comment]
@GabrielBrascher PS. 4.15.1.0 RC1 is out, please help test that :)
